### PR TITLE
Rustfmt const eval and mir interp

### PR DESCRIPTION
**Pull request by oli-obk**

### CODE REVIEW

**Comment by Dylan-DPC** [comment]
rollup=never

File: src/librustc_mir/const_eval.rs [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-    PlaceTy, MPlaceTy, OpTy, ImmTy, Immediate, Scalar, Pointer,
-    RawConst, ConstValue, Machine,
-    InterpResult, InterpErrorInfo, GlobalId, InterpCx, StackPopCleanup, AssertMessage,
-    Allocation, AllocId, MemoryKind, Memory,
-    snapshot, RefTracking, intern_const_alloc_recursive,
+use crate::interpret::{
+    self, intern_const_alloc_recursive, snapshot, AllocId, Allocation, AssertMessage, ConstValue,
+    GlobalId, ImmTy, Immediate, InterpCx, InterpErrorInfo, InterpResult, MPlaceTy, Machine, Memory,
+    MemoryKind, OpTy, PlaceTy, Pointer, RawConst, RefTracking, Scalar, StackPopCleanup,
 };
```
Comment (RalfJung) [inline src/librustc_mir/const_eval.rs]: This will conflict real bad with https://github.com/rust-lang/rust/pull/67327... maybe land that first?

Comment (oli-obk) [reply src/librustc_mir/const_eval.rs]: Yea I guess... I just got so annoyed at autoformatting operating on these files...

**Pull request closed without merging**